open-api/rest-catalog-open-api.py

```diff
@@ -54,6 +54,24 @@ class CatalogConfig(BaseModel):
         ...,
         description='Properties that should be used as default configuration; applied before client configuration.',
     )
+    capabilities: Optional[
+        List[
+            Literal[
+                'pagination',
+                'scan-planning',
+                'views',
+                'vended-credentials',
+                'remote-signing',
+                'oauth2',
+                'sigv4',
```

Comment:
I think we should add `mutli-table-commit` and `register-table` as well since they aren't really required for a catalog implementation.

Reply:
@nastra, looks like we are missing `register-table` still?

Reply:
added